src/AsyncStatusIndicator/AsyncStatusIndicator.tsx

```diff
@@ -27,16 +52,16 @@ export const Props = {
  */
 @skinnable()
 @props(Props)
-export default class AsyncStatusIndicator extends React.PureComponent {
+export default class AsyncStatusIndicator extends React.PureComponent<AsyncStatusIndicatorProps, void> {
```

Comment:
you don't need `void`. `React.PureComponent<AsyncStatusIndicatorProps>` works the same